src/main/java/com/kobylynskyi/graphql/codegen/mapper/ValueMapper.java

```diff
@@ -129,6 +129,9 @@ private String mapEnum(MappingContext mappingContext, EnumValue value, Type<?> g
         if (graphQLType instanceof NonNullType) {
             return mapEnum(mappingContext, value, ((NonNullType) graphQLType).getType());
         }
+        if (graphQLType == null) {
```

Comment:
I think that such null-checks should be done before `if (graphQLType instanceof NonNullType)` and `f (graphQLType instanceof TypeName)`, so can you please move this verification to the top of the method?
Thanks.

Reply:
👍